bin/build-edge

```diff
@@ -12,7 +12,7 @@ bin/build-packages
 
 add_and_commit \
     app/assets/javascripts/pageflow/dist/{react-*,ui,frontend}.js \
-    packages/pageflow/{editor,ui}.js \
+    packages/pageflow/{editor,ui,frontend}.js \
     entry_types/paged/app/assets/javascripts/pageflow_paged/dist/editor.js \
```

Comment:
There is now also `dist/frontend.js`.

Reply:
entry_types/paged/app/assets/javascripts/pageflow_paged/dist/frontend.js?

added in the commit list